.github/workflows/run-standard-tests.yml

```diff
@@ -0,0 +1,194 @@
+---
+name: Run Standard Tests
+# Run minimal end-to-end tests for Khiops, Khiops Coclustering and KNITransfer
+# Tests are lauched on Linux, Windows and macOS ; in serial and in parallel (3 procs)
+# On release, it runs Standard test for binary. Plus MultiTables/SpliceJunction for KNITransfer
+# On debug, it runs only Standard/Iris for Coclustering and KNITransfer and Standard/IrisLight for Khiops
+# In case of errors, test results are available as artifacts for 7 days
+# It is automatically triggered on PR when sources files are modified (files in src directory)
+on:
+  workflow_dispatch:
+  pull_request:
+    paths: [src/**]
+env:
+  KhiopsBatchMode: true
+jobs:
+  # Build the project on each platform for all targets to ensure that all sources and cmake files are correct
+  # The useful binaries are cached for the current run_id. It will be restored only in the same run of the 
+  # build. The cached binaries are delete in the last job (cleanup-cache)
+  build-full-project:
+    strategy:
+      matrix:
+        build-setup:
+          - {os: windows-2022, cmake-preset: windows-msvc}
+          - {os: ubuntu-latest, cmake-preset: linux-gcc}
+          - {os: macos-latest, cmake-preset: macos-clang}
+        config: [debug, release]
+    runs-on: ${{ matrix.build-setup.os }}
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v3
+      - name: Build all binaries
+        uses: ./.github/actions/build-khiops
+        with:
+          preset-name: ${{ matrix.build-setup.cmake-preset }}-${{ matrix.config }}
+          override-flags: -DTESTING=OFF
+      - name: Cache binaries
+        id: cache-binaries-unix
+        uses: actions/cache@v3
+        with:
+          # We add binaries path one by one to avoid *.pdb and other msvc stuffs that generate a cache of 200Mo for windows
+          path: |
+            ${{ github.workspace }}/build/${{ matrix.build-setup.cmake-preset }}-${{ matrix.config }}/bin/MODL
+            ${{ github.workspace }}/build/${{ matrix.build-setup.cmake-preset }}-${{ matrix.config }}/bin/MODL_Coclustering
+            ${{ github.workspace }}/build/${{ matrix.build-setup.cmake-preset }}-${{ matrix.config }}/bin/KNITransfer
+            ${{ github.workspace }}/build/${{ matrix.build-setup.cmake-preset }}-${{ matrix.config }}/bin/MODL.exe
+            ${{ github.workspace }}/build/${{ matrix.build-setup.cmake-preset }}-${{ matrix.config }}/bin/MODL_Coclustering.exe
+            ${{ github.workspace }}/build/${{ matrix.build-setup.cmake-preset }}-${{ matrix.config }}/bin/KNITransfer.exe
+            ${{ github.workspace }}/build/${{ matrix.build-setup.cmake-preset }}-${{ matrix.config }}/bin/KhiopsNativeInterface.dll
+            ${{ github.workspace }}/build/${{ matrix.build-setup.cmake-preset }}-${{ matrix.config }}/lib/libKhiopsNativeInterface.*
+            ${{ github.workspace }}/build/${{ matrix.build-setup.cmake-preset }}-${{ matrix.config }}/lib/KhiopsNativeInterface.lib
```

Comment:
replace with the above defined `${{ env.PRESET_NAME }}`

Reply:
done